projects/gnupg/Dockerfile

```diff
@@ -16,8 +16,9 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER p.antoine@catenacyber.fr
-RUN apt-get update && apt-get install -y make autoconf automake libtool gettext bzip2 gnupg bison flex
+RUN apt-get update && apt-get install -y make autoconf automake libtool gettext bzip2 gnupg bison flex libsqlite3-dev pkg-config
```

Comment:
do you still need `libsqlite3-dev` here?

Reply:
I guess not
But I would rather like this than the manual compiling, don't you think ?

Reply:
aren't lines 20-25 in the build.sh manually compiling and installing a particular version on top of this one?

Reply:
Yes